tutorial-compiler/index.asciidoc

```diff
@@ -0,0 +1,109 @@
+= Tutorial Compiler 
+==== 
+In this scenario you will use the tutorial-compiler to generate the "my-thai-star" tutorial.
+
+The first part will be to setup your workspace.
+
+Afterwards we will show you, how to write your own tutorials using an asciidoc.
```

Comment:
I think we should not mention here that tutorials are written in asciidoc files. We should say it later when we create the file. We should also link the development part of the wiki somewhere.